shared/src/commonMain/kotlin/com/kcc/kmmhackathon/shared/utility/LocationUtil.kt

```diff
@@ -0,0 +1,29 @@
+package com.kcc.kmmhackathon.shared.utility
+
+import kotlin.math.*
+
+typealias Radian = Double
+typealias Degree = Double
+
+class LocationUtil {
+
+    fun getDistance(lat1: Double, lon1: Double,lat2: Double, lon2: Double, inMiles: Boolean): Double {
+
+        // Haversine formula
+        val earthRadius = 3958.75
+        val dLat = (lat1-lat2).toRadian()
+        val dLon = (lon1-lon2).toRadian()
+
+        val a: Double = sin(dLat / 2) * sin(dLat / 2) +
+                cos(lat2.toRadian()) * cos(lat1.toRadian()) *
+                sin(dLon / 2) * sin(dLon / 2);
+        val c: Double = 2 * atan2(sqrt(a), sqrt(1 - a))
```

Comment:
Just curious because these calculations here actually mean nothing to me, is there a reason why it's `a` and `c`?

Reply:
Might be worth adding some small docs around this method, even if it's a link with where we can lookout the method that's being used

Reply:
Yeah, it's based on the Haversine formula will add link to comment.
[Haversine link](https://www.movable-type.co.uk/scripts/latlong.html)

According to the notes on the formula
`c` is the angular distance in radians, and `a` is the square of half the chord length.